pkgs/development/compilers/rust/binaryBuild.nix

```diff
@@ -0,0 +1,106 @@
+{ stdenv, fetchurl, makeWrapper, cacert, zlib, buildRustPackage, curl
+, version
+, src
+, versionType
+, bootstrapping
+}:
+
+let
+  inherit (stdenv.lib) optionalString;
+
+  platform = if stdenv.system == "x86_64-linux"
```

Comment:
this also needs to be defined for all platforms and is defined in multiple places